### ISSUE TITLE
Forum replies - have subject line

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#611]
new site
forum area.

I have noticed that a reply to a post insists on a subject line being included. If you don't put in anything then it lifts the first 20 or so characters from your reply and thinks that is the subject line. Given that you are replying to someone else's subject I am confused as to why a reply would want a news subject line at all?

![image](https://cloud.githubusercontent.com/assets/6060985/6980331/ceacc2b0-d9e7-11e4-9db6-1a72d1a0aaef.png)

P.s. ignore TJBuilders. Tony contributes a lot to the site but invariably negative.

**Comment by pmackay** [comment]
Also something that using Harmony Forum would remove. It doesnt use subject lines.

**Comment by GilesGibson** [comment]
what was the reason we didn't use harmony forum? Was it a non standard one or was it the one that charged?

**Comment by pmackay** [comment]
Its "non standard" but worth investing in IMHO. Core forum isnt great
(which is obvious given how many issues you've raised on it :) ).

Harmony has many improvements but we didnt use it before launch because it
would have required building a different set of migrate code (lots of work)
and we wanted to launch. Plus it was only in dev. However we recently met
the author of Harmony and got some useful info, plus have encouraged a
first release. And I'm writing some migrate code for another project. We're
planning to integrate into Open Local.

**Comment by GilesGibson** [comment]
OK, would like to have a look at it.

However, having got "stuck" with a great forum module on the last site that was developed by one person and never really went that far I would be hesitant to jump into a similar situation. Will it be developed by others as well? What chance that it becomes more common and a "standard"?

**Comment by pmackay** [comment]
There is no chance of it becoming a standard in the sense that the Core
Forum module is a standard because its part of core.

However, listened to http://www.talkingdrupal.com/084 recently, they say
they often dont use Core Forum because it aint that great, and just build a
custom forum using Drupal's building tools.

Also worth thinking, if this site runs for about 5 years, how much future
proofing is feasible? And how much of a tradeoff is that against better
usability?

**Comment by GilesGibson** [comment]
I agree that if we could get a good few years out of it then great. The problem with the previous module was that it could not be upgraded to Drupal 7, the only module on the old site that couldn't be upgraded, hence the need, pretty much forced on us to start again with a new site rather than do the upgrade.

Better functionality would be great as long as it leads to more use.

Where can I see it?

**Comment by pmackay** [comment]
You can see an example install here http://frontend.getharmony.io/. Also
more info at http://www.getharmony.io/.

**Comment by GilesGibson** [comment]
Ah yes, I remember looking at that before. I liked the @ feature to ping
another user. What was the reason we didn¹t go for it? Was it because it was
tough to migrate to? Was it the chargeable one we also looked at?

Happy to go there if it makes it easier to engage people. Would like to look
at a real site though. Demo sites are tough to see how it would work in
reality.

From:  Paul Mackay notifications@github.com
Reply-To:  "folklabs/hernehill.org.uk"
<reply+005c7bb9838d0103ec3cd997b709dc154c8f9cb728903a7492cf0000000111362ece9
2a169ce03f0c931@reply.github.com>
Date:  Friday, 3 April 2015 11:34
To:  "folklabs/hernehill.org.uk" hernehill.org.uk@noreply.github.com
Cc:  Giles Gibson giles.gibson@originaltg.com
Subject:  Re: [hernehill.org.uk] Forum replies - have subject line (#611)

You can see an example install here http://frontend.getharmony.io/. Also
more info at http://www.getharmony.io/.

‹
Reply to this email directly or view it on GitHub
<https://github.com/folklabs/hernehill.org.uk/issues/611#issuecomment-892500
78> .
